apps/design/frontend/src/ballots_screen.tsx

```diff
@@ -238,11 +237,8 @@ function BallotStylesTab(): JSX.Element | null {
                   <Button
                     icon="Done"
                     onPress={() =>
-                      setIsBallotProofingCompleteMutation.mutate(
-                        {
-                          electionId,
-                          finalizedAt: new Date(),
-                        },
```

Comment:
👍 makes sense to remove this as a parameter for easier testing, especially if we don't have a use case to set `finalizedAt` from the frontend